### ISSUE TITLE
apply into webrtc

### ISSUE THREAD

**Issue by fukemy** [banjodayo39/IntelligentCameraWithIOS#1]
Hi, thanks for great lib, did u tried to apply filter into webrtc calling?

**Comment by banjodayo39** [comment]
Hi Kemy,

Sorry for the late reply.
Thanks for finding it helpful, no but if you need assistance I could help
you debug it.

Regards,
Dayo

**Comment by banjodayo39** [comment]
Sorry for the late reply.
Thanks for finding it helpful, no but if you need assistance I could help you debug it.

**Comment by fukemy** [comment]
yeah, thanks, i tried to apply into webrtc, but the filter image is wrong orientation, as you can see here
https://stackoverflow.com/questions/70135988/swift-webrtc-modify-cmsamplebuffer-to-apply-some-filter-effect
